__tests__/index.tests.js

```diff
@@ -158,55 +158,49 @@ describe("GET /api/articles/:article_id", () => {
   });
 });
 
-describe("/api/articles/:article_id/comments", () => {
-  test("GET: 200 responds with an array of comments for a given article id with the correct properties", () => {
-    return request(app)
-      .get("/api/articles/1/comments")
-      .expect(200)
+describe("POST /api/articles/:article_id/comments", () => {
+  test("POST: 201 adds a comment to an article", () => {
+    const newComment = {
+      author: "icellusedkars",
+      body: "hey am I on the right site?",
+    };
+    return request(app)
+      .post("/api/articles/9/comments")
+      .send(newComment)
+      .expect(201)
       .then(({ body }) => {
-        const { comments } = body;
-        expect(comments.length).toBe(11);
-        comments.forEach((comment) => {
-          expect(comment).toHaveProperty("comment_id", expect.any(Number));
-          expect(comment).toHaveProperty("votes", expect.any(Number));
-          expect(comment).toHaveProperty("created_at", expect.any(String));
-          expect(comment).toHaveProperty("author", expect.any(String));
-          expect(comment).toHaveProperty("body", expect.any(String));
-          expect(comment).toHaveProperty("article_id", expect.any(Number));
+        const { comment } = body;
+        expect(comment).toMatchObject({
+          comment_id: 19,
+          body: "hey am I on the right site?",
+          article_id: 9,
+          author: "icellusedkars",
+          votes: 0,
         });
       });
```

Comment:
Please write another test with status 400, where 'newComment' is missing required field(s), e.g. no username or body properties.